### ISSUE TITLE
Triggers ignore modifier keys

### ISSUE THREAD

**Issue by dfaught** [kiibohd/controller#217]

**Comment by dfaught** [comment]
It also appears there are some issues with combinations in results:

**U"C" : U"CTRL" + U"SPACE", U"C";**
I would expect the above result combination to send a combination of ctrl + space, followed by a 'c' the actual result sent is ctrl+space followed by ctrl+c.  It appears the ctrl is being carried through.

**Comment by InfinityTotality** [comment]
Yeah, sequences don't release any of the keys until the very end of the sequence for the moment
